doc/amazon_sagemaker_operators_for_kubernetes_jobs.rst

```diff
@@ -304,6 +304,134 @@ job stops or completes.
 continue to show on the Amazon SageMaker console. The delete command
 takes about 2 minutes to clean up the resources from Amazon SageMaker.
 
+Sagemaker Debugger Jobs
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Create training job has an option to run asynchronous debugger job.
```

Comment:
`When creating a SageMaker training job, you have an option to run asynchronous debugger jobs for your model`?